### DISCUSSION TITLE
Nonsteamlauncher messed up the whole Deck

### DISCUSSION THREAD

**Discussion by Caki883** [moraroy/NonSteamLaunchers-On-Steam-Deck#324] [Q&A]
Installed battle.net and Epic , didn´t liked it and Uninnstalled them. Now all the thousands of nonsteam games are gone  and the funny part is that epic and Battle.net can´t be deleted from the shortcuts. They always come back after restart

**Comment by moraroy** [comment]
@Caki883 i think youre the same person from the discord, but your library is just gone, not your games. they are backed up in a .bak file

**Comment by Caki883** [comment]
Yes 👍, but the Steam Rom manager says it is damadged. Need to look deeper
in the Shortcut.vfd

Roy ***@***.***> schrieb am Mo., 17. Juni 2024, 06:55:

**Reply by moraroy** [reply]
@Caki883 dang :/ ok initially, we're you using the plugin version or the desktop version of NonSteamLaunchers?

**Comment by Caki883** [comment]
.sh

Roy ***@***.***> schrieb am Mo., 17. Juni 2024, 07:28:

**Comment by moraroy** [comment]
@Caki883 this has since been fixed correct?
